base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
+  min-width: 414px;
+  height: 100vh;
+  background-color: RGB(25, 30, 40);
+  display: flex;
+  flex-direction: column;
+}
+
+.navigationBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 400px;
+  height: 75px;
+  transition-duration: 5s;
+}
+
+.navigationBarToggle {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  background-color: rgba(200, 200, 200, 0.1);
+  width: 400px;
+  height: 75px;
+
+  transition-duration: 5s;
+}
+.mainArea {
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  background-color: transparent;
+
+  margin-left: 2%;
+}
+
+.menuIcon {
+  float: right;
+  margin: 3%;
+  color: seashell;
+  font-size: 270%;
+  opacity: 70%;
+  transition-duration: 0.7s;
+  text-shadow: 1pt 1pt rgba(200, 200, 200, 0.4);
+}
+
+.menuIconToggle {
+  opacity: 0%;
+  transition-duration: 0.7s;
+}
+
+.navButton {
+  opacity: 0%;
+  width: 70px;
+  height: 30px;
```

Comment:
These fixed size rules will greatly limit the `font-size` and amount of characters of text or any content of elements using this class. Try to make it a bit more flexible with `padding` for example, or having the parent element assigning the size

Reply:
!!! Change planed in dedicated user story for UI responsiveness

Thank you